src/rest/template.pug

```diff
@@ -45,41 +46,44 @@ include ./resource.pug
 
         include ../snippets/help.pug
         h2 Endpoints
-        for resource in rest.resources
-            .panel.panel-default.panel-rest
-                a(name=resource.uniqueId id=resource.uniqueId)
-                a.panel-reference.icon.icon-link-1(href=`#${resource.uniqueId}`)
-                .panel-heading
-                    h3.panel-title= resource.displayName || resource.relativeUri
-                .panel-body
-                    if resource.description
-                        .top-resource-description!= marked(resource.description)
-                    .panel-group
-                        +resourceModals(resource)
-                        table(style='width:100%')
-                            tbody
-                                +resourceRows(resource)
+        if rest.resources
+            for resource in rest.resources
```

Comment:
I'm not sure if I like this, I feel like it would be cleaner to define these as empty arrays in the fallback object.